src/utils/generator/ReplicaFiller.cpp

```diff
@@ -149,7 +149,7 @@ void ReplicaFiller::init() {
 #endif
 	Domain domain(0, nullptr);
 	_inputReader->readPhaseSpaceHeader(&domain, 0.0);
-	_inputReader->readPhaseSpace(&basisContainer, &lmu, &domain, &domainDecomp);
+    _inputReader->readPhaseSpace(&basisContainer, &domain, &domainDecomp);
```

Comment:
lmu was a local object this time, check if you can delete line 144. 
spaces vs tabs